ui/js/controllers/jobs.js

```diff
@@ -106,28 +106,6 @@ treeherderApp.controller('ResultSetCtrl', [
         $scope.getCountText = function(resultStatus) {
             return thResultStatusInfo(resultStatus).countText;
         };
-        $scope.viewJob = function(job) {
```

Comment:
You are right, can't believe I missed that! I'll put that function back in (and re-enable the test).